nav2_controller/src/controller_server.cpp

```diff
@@ -242,7 +242,10 @@ ControllerServer::on_activate(const rclcpp_lifecycle::State & /*state*/)
 {
   RCLCPP_INFO(get_logger(), "Activating");
 
-  costmap_ros_->activate();
+  const auto costmap_ros_state = costmap_ros_->activate();
```

Comment:
For planner server too?

Reply:
I haven't considered the planner server, it should get the same change.